scripts/build-image.sh

```diff
@@ -105,7 +105,9 @@ run_buildx() {
     build_args+=("${root_dir}")
   fi
   if [ "${do_test}" = "true" ] ; then
-    docker buildx build --target=test "${build_args[@]}"
+    if ! docker buildx build --target=test "${build_args[@]}" ; then
```

Comment:
Not a change request, but I wonder, does isn't running with `errexit` set enough to exit on `docker buildx` failure?

Reply:
I also thought that, I was suspecting that it could be due to this function being called inside an if-clause (`if ! run_buildx ; then`). I'd rather not worry about it too much right now, another reason for #10 to happen sooner!